.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+## Proposed changes
+
+Describe your changes here. Why are they necessary? If it fixes a bug or
```

Comment:
This is more of a stylistic request, but what do you think of changing this block (as well as the one in `Justification of...` and `Further Comments` to be italicized to more easily discern that they are prompts that are meant to be deleted.